src/components/ArticleCard/ArticleCard.spec.js

```diff
@@ -0,0 +1,281 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import ArticleCard from './index';
+import { articleCardMockData } from '../../../__mocks__/mockData';
+
+const {
+  title,
+  description,
+  body,
+  image,
+  authorImage,
+  publishedDate,
+  author,
+  readTime
+} = articleCardMockData;
+
+describe('ArticleCard', () => {
+  let articleCard;
+
+  beforeEach(() => {
+    articleCard = mount(
+      <BrowserRouter>
+        <ArticleCard
+          title={title}
+          description={description}
+          body={body}
+          image={image}
+          authorImage={authorImage}
+          publishedDate={publishedDate}
+          author={author}
+          readTime={readTime}
+        />
+      </BrowserRouter>
+    );
+  });
+
+  it('renders the article card successfully', () => {
+    expect(articleCard).toBeDefined();
+    expect(articleCard.find('div').length).toBe(4);
+    expect(articleCard.find('img').length).toBe(2);
+    expect(articleCard.find('h1').length).toBe(1);
+    expect(articleCard.find('p').length).toBe(3);
+    expect(articleCard.find('span').length).toBe(3);
+    expect(articleCard.find('button').length).toBe(1);
+  });
+
+  it('should render an article card with a title', () => {
+    articleCard = mount(
+      <BrowserRouter>
+        <ArticleCard
```

Comment:
Article card should accept `article`, `showBookmark`, `isBookmarked` and `toggleBookmark`